ghostwriter/reporting/templates/reporting/report_detail.html

```diff
@@ -144,6 +145,21 @@ <h4>Current Findings</h4>
                                         title="Click-n-drag to reposition or categorize"
                                     ></td>
                                     <td class="align-middle"><a id="delete-target-content-finding-{{ finding.id }}" class="clickable" href="{% url 'reporting:local_edit' finding.id %}">{{ finding.title }}</a></td>
+                                    <td class="neutral align-middle" style="color:
```

Comment:
Colors for severity titles should be defined in the CSS files. That makes them consistent across all pages and more easily customizable. A lot of colors like this have been declared here:

https://github.com/GhostManager/Ghostwriter/blob/master/ghostwriter/static/css/base_styles.css#L82